### DISCUSSION TITLE
Have daily challenge beatmaps viewable on the site

### DISCUSSION THREAD

**Discussion by lan641** [ppy/osu#29123] [Ideas] [answered]
It'd be more convenient to see what the current daily challenge map is on the site than opening lazer. I imagined it like you hover over the daily challenge box in your profile that directs you a link to a page dedicated to daily challenge with leaderboards, stats, and stuff. It would be a nice feature to have.

**Comment by peppy** [comment, marked as answer]
I've tasked @nanaya with making this happen!
